plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/model/JDK.java

```diff
@@ -24,10 +24,10 @@ public enum JDK {
      * Available JDKs
      * See <a href="https://www.jenkins.io/doc/book/platform-information/support-policy-java/">Java Support Policy</a> for details
      */
-    JAVA_8(8, true, null, "2.346.1"),
-    JAVA_11(11, true, "2.164.1", "2.462.3"),
-    JAVA_17(17, true, "2.346.1", null),
-    JAVA_21(21, true, "2.426.1", null);
+    JAVA_8(8, true, null, "2.346.1", "2042.v787a_641a_9b_26"),
```

Comment:
```suggestion
    JAVA_8(8, true, null, "2.346.1", "1900.v9e128c991ef4"),
```

See my latest comment